server/datastore/mysql/migrations/tables/20241209164540_AddSecretVariablesTable.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241209164540, Down_20241209164540)
+}
+
+func Up_20241209164540(tx *sql.Tx) error {
+
+	if tableExists(tx, "secret_variables") {
+		return nil
+	}
+
+	_, err := tx.Exec(`
+		CREATE TABLE secret_variables (
+		id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+		name VARCHAR(255) NOT NULL,
+		value BLOB NOT NULL,
+		created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
```

Comment:
Oh! I always forget to add the `(6)` to have milliseconds, I'll update the timestamps in the scoped labels story.